.github/workflows/images.yml

```diff
@@ -26,8 +26,16 @@ jobs:
         run: |
           images=""
           for n in $name; do
-            tags=$(curl -su ":" https://containers.intersystems.com/v2/intersystems/${n}/tags/list | jq -r '.tags[]' | awk '!/(-linux)|([1-4]-preview)/' | awk '!/\.[1-4]\.0\./' | uniq)
-            for tag in $tags; do images+='"'${n}:${tag}'",'; done
+            tags=$(curl -su ":" https://containers.intersystems.com/v2/intersystems/${n}/tags/list | jq -r '.tags[]' | awk '!/(-linux)|([1-4]-preview)|(-em)|(-cd)/' | awk '!/\.[1-4]\./' | uniq)
```

Comment:
Maybe add a `| sort` before `| uniq` in case future API doesn't return sorted tags?

Reply:
Good idea!